diff -uNr ftjam-2.5.2/execunix.c ftjam-2.5.2_patched/execunix.c
--- ftjam-2.5.2/execunix.c	2006-06-05 12:52:27.000000000 +0200
+++ ftjam-2.5.2_patched/execunix.c	2022-07-15 11:23:46.252330041 +0200
@@ -38,6 +38,12 @@
  * 12/27/02 (seiwald) - grist .bat file with pid for system uniqueness
  */
 
+# include <sys/types.h>
+# include <unistd.h>
+#if !defined(__MINGW32__) && !defined(__MINGW64__)
+#         include <wait.h>
+#endif
+
 # include "jam.h"
 # include "lists.h"
 # include "execcmd.h"
diff -uNr ftjam-2.5.2/fileunix.c ftjam-2.5.2_patched/fileunix.c
--- ftjam-2.5.2/fileunix.c	2006-06-05 12:52:07.000000000 +0200
+++ ftjam-2.5.2_patched/fileunix.c	2022-07-15 11:06:21.988108324 +0200
@@ -33,6 +33,8 @@
  * 12/30/02 (seiwald) - skip solaris' empty archive member names (/, //xxx)
  */
 
+# include <unistd.h>
+
 # include "jam.h"
 # include "filesys.h"
 # include "pathsys.h"
diff -uNr ftjam-2.5.2/make1.c ftjam-2.5.2_patched/make1.c
--- ftjam-2.5.2/make1.c	2006-06-05 12:52:04.000000000 +0200
+++ ftjam-2.5.2_patched/make1.c	2022-07-15 11:06:37.264109594 +0200
@@ -49,6 +49,8 @@
  * 12/03/02 (seiwald) - fix odd includes support by grafting them onto depends
  */
 
+# include <unistd.h>
+
 # include "jam.h"
 
 # include "lists.h"
